wfe2/verify.go

```diff
@@ -376,6 +376,36 @@ func (wfe *WebFrontEndImpl) extractJWK(jws *jose.JSONWebSignature) (*jose.JSONWe
 	return key, nil
 }
 
+// acctIDFromURL extracts the numeric int64 account ID from a ACMEv1 or ACMEv2
+// account URL. If the acctURL has an invalid URL or the account ID in the
+// acctURL is non-numeric a malformed problem is returned.
```

Comment:
Let's say "MalformedProblem" here to make it clear we're talking about a data type, as opposed to describing the returned problem as malformed.

Reply:
Sounds good. Fixed in f854a81